src/routes/auth/refresh/server.test.ts

```diff
@@ -0,0 +1,49 @@
+import { describe, it, expect, vi } from 'vitest';
+import { COOKIE_SESSION_NAME } from '../constants';
+import { POST } from './+server';
+
+vi.mock('@sveltejs/kit', () => ({
+  json: vi.fn((data) => data),
+}));
+
+describe('POST function', () => {
+  it('should set the cookie if token is provided', async () => {
+    const mE = {
+      request: {
+        json: vi.fn().mockResolvedValue({
+          [COOKIE_SESSION_NAME]: 'test-token',
+        }),
+      },
+      cookies: {
+        set: vi.fn(),
+        delete: vi.fn(),
+      },
+    };
+
+    const response = await POST(mE);
+
+    expect(mE.request.json).toHaveBeenCalled();
+    expect(mE.cookies.set).toHaveBeenCalledWith(COOKIE_SESSION_NAME, 'test-token', { path: '/' });
```

Comment:
This line is too long. Make it shorter. 
```suggestion
    expect(mE.cookies.set).toHaveBeenCalledWith(COOKIE_SESSION_NAME, 'test-token', { path: '/' });
```